pkg/wkspace/minimal.go

```diff
@@ -91,13 +93,16 @@ func (m *MinimalWorkspace) BounceHelm(extraArgs ...string) error {
 	if err == nil {
 		defer func(oldpath, newpath string) {
 			_ = os.Rename(oldpath, newpath)
-		}(backup, pathing.SanitizeFilepath(filepath.Join(path, "values.yaml")))
+		}(backup, pathing.SanitizeFilepath(filepath.Join(path, valuesYaml)))
```

Comment:
I'm pretty sure this should be the `default-values.yaml` file here. This is because at deployment time we have the ability to add values from terraform, but since those can change this is templated during the deploy stage. For example if the values file contains `example: '{{ .Import.Terraform.access_key_id }}'`. The workflow looks as follows:
- create a backup of the values file
- template the value from terraform into the values file so it conains `example: value`
- restore the backed up values file so that it contains `example: '{{ .Import.Terraform.access_key_id }}'` again and can be templated again during the next deployment (as the terraform output can change).